### DISCUSSION TITLE
<title> I have a problem whit my project page

### DISCUSSION THREAD

**Discussion by Shinnizlex** [community/community#49853] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs

I try to make my own portfolio page but i have this error. 

https://github.com/Shinnizlex/Shinnizlex.github.io

**Comment by bbkx226** [comment, marked as answer]
Based on the error message you provided, it seems that Jekyll is unable to find the file "style.scss" in the "assets/css/" directory. Double-check to make sure that the file exists in the correct directory.
